### PR TITLE
feat!: migrate API to microgenerator

### PR DESCRIPTION
**Pull request by danoscarmike**

### CODE REVIEW

**Comment by danoscarmike** [comment]
Flipping CLA to yes as this PR includes commits transferred from another org/repo.

File: UPGRADING.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+```sh
+$ fixup_translation_v3_keywords.py --input-directory .samples/ --output-directory samples/
+```
+
+**Before:**
+```py
+from google.cloud import translate
+
+client = translate.TranslationServiceClient()
+parent = client.location_path(<PROJECT_ID>, <LOCATION>)
```
Comment (busunkim96) [inline UPGRADING.md:38]:
```suggestion
parent = client.location_path("<PROJECT_ID>", "<LOCATION>")
```

File: UPGRADING.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+response = client.translate_text(
+    parent=parent,
+    contents=[text],
```
Comment (busunkim96) [inline UPGRADING.md:42]: Could you declare the variable and set it to some value before the call?

```py
text = "Good morning!"

response = client.translate_text(...

```

Comment (danoscarmike) [reply UPGRADING.md:43]: Done!

**Pull request merged**